packages/zowe-explorer/src/dataset/actions.ts

```diff
@@ -1063,18 +1063,16 @@ export async function submitJcl(datasetProvider: api.IZoweTree<api.IZoweDatasetT
 /**
  * Shows a confirmation dialog (if needed) when submitting a job.
  *
- * @param node The node/member that is being submitted
+ * @param nodeOrFileName The node/member that is being submitted, or the filename to submit
  * @param ownsJob Whether the current user profile owns this job
- * @param fileName When submitting local JCL, use the document file name to submit
  * @returns Whether the job submission should continue.
  */
-export async function confirmJobSubmission(node: api.IZoweTreeNode, ownsJob: boolean, fileName?: string): Promise<boolean> {
+export async function confirmJobSubmission(nodeOrFileName: api.IZoweTreeNode | string, ownsJob: boolean): Promise<boolean> {
```

Comment:
I always get worried when we change the signature of an exported function. 😨 
But if others are ok with this, I think it's fine 😋

Reply:
I feel you @zFernand0 
I think this will be fine since it isn't extender facing or part of the ZE API package.